File: LeetCode/Easy/0867-transpose-matrix/0867-transpose-matrix.cpp
class Solution {
public:
    vector<vector<int>> transpose(vector<vector<int>>& matrix) {
        int n = matrix.size();
        int m = matrix[0].size();
        vector<vector<int>> ret(m, vector<int>(n));
        
        for (int i = 0; i < m; ++i)
            for (int j = 0; j < n; ++j)
                ret[i][j] = matrix[j][i];
        
        return ret;
    }
};

File: LeetCode/Easy/0867-transpose-matrix/0867-transpose-matrix.go
func transpose(matrix [][]int) [][]int {
    n := len(matrix)
    m := len(matrix[0])
    ret := make([][]int, m)

    for i := 0; i < m; i++ {
        ret[i] = make([]int, n)
        for j := 0; j < n; j++ {
            ret[i][j] = matrix[j][i]
        }
    }
    
    return ret
}